src/prompts/followUp/employees

```diff
@@ -0,0 +1,17 @@
+const employees = `
+Extract the total number of employees that the company had in the given period. Add it as field employees. In the report, this can be referred to as Average value of employees. Make sure that it is the total number of employees, not a subset such as one gender or employees in only one country. Extract this data for all available years. 
```

Comment:
I assume "Average value of employees" is an intentional word choice to match the reports?